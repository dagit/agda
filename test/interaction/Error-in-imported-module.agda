module Error-in-imported-module where

open import Error-in-imported-module.M
