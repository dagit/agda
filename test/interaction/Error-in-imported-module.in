ioTCM currentFile NonInteractive (cmd_load currentFile [])
