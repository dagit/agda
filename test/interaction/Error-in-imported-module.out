(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Error-in-imported-module (Error-in-imported-module.agda).\n" t)
(agda2-info-action "*Type-checking*" " Checking Error-in-imported-module.M (Error-in-imported-module/M.agda).\n" t)
(agda2-info-action "*Error*" "Error-in-imported-module/M.agda:4,7-10\nSet‚ÇÅ != Set\nwhen checking that the expression Set has type Set" nil)
(agda2-goto '("Error-in-imported-module/M.agda" . 33))
(agda2-status-action "")
